RoutingKit/RoutingKit/_Router/MappedRouter.swift

```diff
@@ -284,20 +280,13 @@ open class MappedRouter: NSObject, RouterProtocol, ParsingProtocol, CombineObser
         } else {
             let transformed = transform(request: request)
             if let map = self.map(for: transformed) {
-                let previousTracking = previousTracking()
                 backtrack(request: transformed, animated: animated) { [weak self] data, completed in
                     if completed {
-                        if let viewController = data as? TrackingViewProtocol {
-                            viewController.logView(path: transformed.path, data: nil, from: previousTracking?.path, time: previousTracking?.startTime)
-                        }
                         completion?(nil, true)
                     } else {
                         self?.route(dependencies: map, request: transformed, completion: { [weak self] _, successful in
                             if successful {
                                 self?.navigate(to: map, request: transformed, presentation: presentation ?? transformed.presentation, animated: animated, completion: { /* [weak self] */ data, successful in
-                                    if successful, let viewController = data as? TrackingViewProtocol {
-                                        viewController.logView(path: transformed.path, data: nil, from: previousTracking?.path, time: previousTracking?.startTime)
-                                    }
```

Comment:
we were logging a view event both here and in the `viewDidAppear` lifecycle method which resulted in double logging. Removed logging from within Routing Kit in favor of `TrackingViewController` lmk if u disagree with this decision or if i am missing something